spec/version_requirement_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe MetadataJsonLint::VersionRequirement do
+  describe '.new' do
+    it { expect(described_class.new('')).to be_a(MetadataJsonLint::VersionRequirement) }
+    it { expect(described_class.new('>= 1.0')).to be_a(MetadataJsonLint::VersionRequirement) }
+    it { expect { described_class.new('## 1.0') }.to raise_error(ArgumentError) }
+  end
+
+  describe '#mixed_syntax?' do
+    it { expect(described_class.new('>= 1.0.0').mixed_syntax?).to be false }
+    it { expect(described_class.new('1.0.x').mixed_syntax?).to be false }
```

Comment:
Hrm, then why does the mixed_version_syntax test produce a warning with `3.x` without any second operator?